config/jobs/periodic/knative/operator/main/operator-main.gen.yaml

```diff
@@ -0,0 +1,66 @@
+periodics:
+  - name: knative-operator-main-periodic
+    labels:
+      preset-knative-powervs: "true"
+    decorate: true
+    cron: "20 0 * * *"
+    extra_refs:
+      - base_ref: main
+        org: valen-mascarenhas14
+        repo: knative-upstream-ci
+        workdir: true
+      - base_ref: main
+        org: knative
+        repo: operator
+        workdir: false
+    spec:
+      containers:
+        - image: quay.io/powercloud/knative-prow-tests:latest
+          resources:
+            requests:
+              cpu: "4000m"
+            limits:
+              cpu: "4000m"
```

Comment:
@valen-mascarenhas14 Any reference/analysis done on why 4000m Guaranteed CPUs needed?

Reply:
Modified it to 1500m